packages/experimental/Menu/src/MenuTrigger/useMenuTrigger.ts

```diff
@@ -10,15 +11,26 @@ export const useMenuTrigger = (_props: MenuTriggerProps) => {
   const openOnHover = context.openOnHover;
   const triggerRef = context.triggerRef;
 
+  const delayHover = Platform.select({
+    macos: 100,
+    default: 500, // win32
+  });
```

Comment:
These delays were for when a submenu is opened off a menu, not for the main menu. I don't know we'd always want the delay for the main menu on macOS. Is that a windows / win32 requirement?

Reply:
There's a short delay for opening a menu on hover in win32 though it was for submenus so I don't know if it's a strict requirement. Let me ask PM.

Reply:
I asked if there was any example of an "openOnHover" scenari for a ContextMenu on macOS, and we didn't know of any. So at least on macOS, we don't have an opinion / prior art to make an opinion. Since this delay will (presumably?) be reused for submenu, I guess it's OK

Reply:
win32 doesn't really have a specific scenario for it either. it was part of the spec, so in it went 🤷🏻‍♀️

Anyway PM says that it'd probably be useful to have so we'll add it for now. We can make it win32 only if we need to.

Reply:
Eh.. I feel like there's a difference between "This is not supported on the platform" (engineering) vs "This is not a platform paradigm" (design). I personally prefer to mark props as win32/macOS only if they truly aren't supported. So I think it's OK.